docs/docsite/rst/community/development_process.rst

```diff
@@ -23,6 +23,62 @@ Ansible accepts code via **pull requests** ("PRs" for short). GitHub provides a
 
 Because Ansible receives many pull requests, we use an automated process to help us through the process of reviewing and merging pull requests. That process is managed by **Ansibullbot**.
 
+Backport Pull Request Process
+-----------------------------
+
+After the pull request submitted to Ansible for the ``devel`` branch is
+accepted and merged, the following instructions will help you create a
+pull request to backport the change to a previous stable branch.
+
+.. note::
+
+    These instructions assume that ``stable-2.5`` is the targeted release
+    branch for the backport.
+
+.. note::
+
+    These instructions assume that ``https://github.com/ansible/ansible.git``
+    is configured as a ``git remote`` named ``upstream``. If you do not use
+    a ``git remote`` named ``upstream`` adjust the instructions accordingly.
+
+#. Prepare your devel, stable, and feature branches:
+
+   ::
+
+       git checkout devel
+       git --fetch upstream
+       git merge upstream/devel
+       git checkout stable-2.5
+       git pull
+       git checkout -b cherry-pick/2.5/[PR_NUMBER_FROM_DEVEL]
+
+#. Cherry pick the relevant commit SHA from the devel branch into your feature
+   branch, handling merge conflicts as necessary:
+
+   ::
+
+       git cherry-pick -x [SHA_FROM_DEVEL]
+
+#. Add a changelog entry for the change, and commit it.
+
+#. Push your feature branch to your fork on GitHub:
+
+   ::
+
+       git push origin cherry-pick/2.5/[PR_NUMBER_FROM_DEVEL]
+
+#. Submit the pull request for ``cherry-pick/2.5/[PR_NUMBER_FROM_DEVEL]``
+   against the ``stable-2.5`` branch
+
+.. note::
+
+    The choice to use ``cherry-pick/2.5/[PR_NUMBER_FROM_DEVEL]`` as the
```

Comment:
bikeshed: i would recommend 'backport' instead of cherry-pick as sometimes the PR will not be possible to be made by cherry-picking (too much code changed).